packages/non-deployable/token-allocation/README.md

```diff
@@ -11,13 +11,18 @@ To be functional at least one of the following underyling adapter locations will
 | Required? |            Name             |                   Description                    | Options | Defaults to |
 | :-------: | :-------------------------: | :----------------------------------------------: | :-----: | :---------: |
 |           |   `AMBERDATA_ADAPTER_URL`   |  The location of an Amberdata external adapter   |         |             |
+|           | `CFBENCHMARKS_ADAPTER_URL`  | The location of a CFBenchmarks external adapter  |         |             |
 |           |    `COINAPI_ADAPTER_URL`    |    The location of a CoinAPI external adapter    |         |             |
 |           |   `COINGECKO_ADAPTER_URL`   |   The location of a CoinGecko external adapter   |         |             |
 |           | `COINMARKETCAP_ADAPTER_URL` | The location of a CoinMarketCap external adapter |         |             |
+|           | `COINMETRICS_ADAPTER_URL  ` |  The location of a CoinMetrics external adapter  |         |             |
 |           |  `COINPAPRIKA_ADAPTER_URL`  |  The location of a CoinPaprika external adapter  |         |             |
 |           |  `COINRANKING_ADAPTER_URL`  |  The location of a CoinRanking external adapter  |         |             |
 |           | `CRYPTOCOMPARE_ADAPTER_URL` | The location of a CryptoCompare external adapter |         |             |
+|           |    `FINAGE_ADAPTER_URL`     |    The location of a Finage external adapter     |         |             |
 |           |     `KAIKO_ADAPTER_URL`     |     The location of a Kaiko external adapter     |         |             |
+|           |    `NCFX_ADAPTER_URL  `     |     The location of a NCFX external adapter      |         |             |
+|           |    `NOMICS_ADAPTER_URL`     |    The location of a Nomics external adapter     |         |             |
```

Comment:
Is `nomics` actually supported by TA ? Can't seem to find it in the config

Reply:
Yikes, I was working off of `develop` and Nomics had been deleted since then. Good catch, updating!